natvis/object_visualizer.h

```diff
@@ -20,6 +20,12 @@ enum class PropertyCategory
     Class,
 };
 
+enum class ObjectType
```

Comment:
this change from bool can be ignored - thought I'd need a tri-state for implementations, but just decided to keep it.

the only real meat is winrt::impl::producer addition and removing the condition on expanding (which permits raw data to always be visible)